app/css/lab-orders-list.scss

```diff
@@ -46,3 +54,84 @@
     max-width: 90px;
 }
 
+.name-emrid-order-filter {
+  display: flex;
+  width: 75%;
+  position: relative;
+
+  input {
```

Comment:
@jomadoye so if you notice, this particular input is nested in the `.name-emrid-order-filter` class so it will only apply for only `<input />` elements under that class which is just one. I think that's unique enough.